proxy/pkg/zdmproxy/clienthandler.go

```diff
@@ -1434,10 +1435,12 @@ func (ch *ClientHandler) executeRequest(
 		log.Tracef("Forwarding request with opcode %v for stream %v to %v",
 			f.Header.OpCode, f.Header.StreamId, common.ClusterTypeOrigin)
 		ch.originCassandraConnector.sendRequestToCluster(originRequest)
+		ch.targetCassandraConnector.sendHeartbeat(ch.startupRequest.Header.Version, ch.conf.HeartbeatIntervalMs)
 	case forwardToTarget:
 		log.Tracef("Forwarding request with opcode %v for stream %v to %v",
 			f.Header.OpCode, f.Header.StreamId, common.ClusterTypeTarget)
 		ch.targetCassandraConnector.sendRequestToCluster(targetRequest)
+		ch.originCassandraConnector.sendHeartbeat(ch.startupRequest.Header.Version, ch.conf.HeartbeatIntervalMs)
```

Comment:
just leaving a note here so I don't forget to check this in a future review:

make startup threadsafe (doesn't have to be a mutex, an atomic value is probably fine)

Reply:
@joao-r-reis Alright, I turned the `startupRequest` var into an atomic value.